module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Set-EntraUserManager.md

```diff
@@ -25,41 +25,36 @@ Updates a user's manager.
 
 ```powershell
 Set-EntraUserManager 
- -ObjectId <String> 
+ -UserId <String> 
  -RefObjectId <String> 
  [<CommonParameters>]
 ```
 
 ## Description
 
-The `Set-EntraUserManager` cmdlet update the manager for a user in Microsoft Entra ID. Specify the `ObjectId` and `RefObjectId` parameters to update the manager for a user in Microsoft Entra ID.
+The `Set-EntraUserManager` cmdlet update the manager for a user in Microsoft Entra ID. Specify the `UserId` and `RefObjectId` parameters to update the manager for a user in Microsoft Entra ID.
 
 ## Examples
 
 ### Example 1: Update a user's manager
 
 ```powershell
 Connect-Entra -Scopes 'User.ReadWrite.All'
-$manager = Get-EntraUser -ObjectId 'Manager@contoso.com'
-$params = @{
-    ObjectId = 'SawyerM@contoso.com'
-    RefObjectId = $manager.ObjectId
-}
-Set-EntraUserManager @params
+Set-EntraUserManager -UserId 'aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb' -RefObjectId 'eeeeeeee-4444-5555-6666-ffffffffffff'
```

Comment:
keep existing example as it is only change objectid specific changes

Reply:
updated.